.ebextensions/sshd.config

```diff
@@ -0,0 +1,21 @@
+packages:
+  yum:
+    python27-devel: []
```

Comment:
is this python 2.7x reference significant?

Reply:
It's Elastic beanstalk's default configuration. Not needed anymore.